melange.rsa

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
this shouldn’t be here, or the public key :)

Reply:
Maybe there's a setting or GH action we could use to check for this at some point 🤔  (e.g. gitleaks)

Reply:
We can add these paths to .gitignore, and grep for PRIVATE KEY in lint.sh